packages/util-create-request/README.md

```diff
@@ -0,0 +1,70 @@
+# @aws-sdk/util-create-request
+
+This package provides function to create request object from given client and command. 
+You can supply either Node client or browser client. A common use case for it can be
+generating request object and then supply to presigners to create presigned url.
+
+When calling the `createRequest()`, the `initialize` and `serialize` middlewares 
+from both client and command are extracted and resolved into a handler. This handler
+will return a promise of `HttpRequest` object. So any modifications happen in `build`
+and `finalize` middleware won't be reflected to generated `httpRequest` object. For
+example, the `Content-Length` header won't be included in the result.
+
+JavaScript Examples: 
+
+```javascript
+const S3Client = require('@aws-sdk/client-s3-node/S3Client').S3Client;
+const GetObject = require('@aws-sdk/client-s3-node/commands/GetObjectCommand').GetObjectCommand;
```

Comment:
import `createRequest` from `@aws-sdk/util-create-request`

ditto for typescript examples